don't always rebuild this header as it won't change between compile & install

--- a/SConstruct
+++ b/SConstruct
@@ -1174,7 +1174,6 @@
     f.close()
 timebase = env.Command(target="timebase.h",
                        source=["leapseconds.cache"], action=timebase_h)
-env.AlwaysBuild(timebase)
 
 env.Textfile(target="gpsd_config.h", source=confdefs)
 env.Textfile(target="gpsd.h", source=[File("gpsd.h-head"), File("gpsd_config.h"), File("gpsd.h-tail")])
